ansible/deploy-all.yaml

```diff
@@ -35,31 +35,31 @@
 # Services
 ############################################
   
-- include: playbooks/10-iplant-data-cfg.yaml
-- include: playbooks/10-jex.yaml
-- include: playbooks/10-services-cfg.yaml
+- include: playbooks/de-iplant-data-cfg.yaml
+- include: playbooks/de-jex.yaml
+- include: playbooks/de-services-cfg.yaml
 
 ############################################
 # DE UI
 ############################################
-- include: playbooks/10-ui.yaml
+- include: playbooks/de-ui.yaml
 
 ############################################
 # Logging; ELK Stack
 ############################################
-- include: elk-stack.yaml
+- include: support-elk-stack.yaml
 
-- include: playbooks/rsyslog-config.yaml
-- include: playbooks/10-logstash-forwarder.yaml
+- include: playbooks/support-rsyslog-config.yaml
+- include: playbooks/support-logstash-forwarder.yaml
 
 ############################################
 # Systems
 ############################################
-- include: systems.yaml
+- include: infra-systems.yaml
```

Comment:
These `infra-*` and `support-*` playbooks were moved to our internal repo, so I don't think they can be called from this playbook anymore.